life/modernized/20-chapter-20.adoc

```diff
@@ -173,14 +183,17 @@ In the course of this year, a work was given to the world,
 purporting to explain the opinions of the Quakers,
 by an author who chose to conceal his name.
 This brought William Penn again before the public, in a tract, entitled,
-"`A brief answer to a false and foolish libel.`"
+_A brief Answer to a False and Foolish Libel._
 Of this answer it is unnecessary to present any analysis,
 as it consists chiefly of returns to misrepresentations
 and perversions of the opinions of Friends.
 The conclusion manifesting the readiness of the author to answer
 all reasonable inquiries respecting his faith and principles,
 is subjoined.
 
+[quote]
```

Comment:
Since this is quoting from one of Penn's written tracts, I put it in block quotes. I wasn't sure if that was right, if that's only reserved for writing that is NOT from Quakers. 😬

Reply:
looks fine to me 🙂